packages/x-codemod/src/v7.0.0/pickers/preset-safe/index.ts

```diff
@@ -1,9 +1,11 @@
 import transformRenameComponentsToSlots from '../../../v6.0.0/pickers/rename-components-to-slots';
+import transformRenameDefaultCalendarMonthReferenceDate from '../rename-default-calendar-month-reference-date';
```

Comment:
Nitpick: Having the `to` before the reference date wouldn't hurt both here and in the folder name. 🙈